src/libraries/System.Security.Cryptography.Xml/ref/System.Security.Cryptography.Xml.cs

```diff
@@ -11,8 +11,10 @@ public sealed partial class CipherData
         public CipherData() { }
         public CipherData(byte[] cipherValue) { }
         public CipherData(System.Security.Cryptography.Xml.CipherReference cipherReference) { }
-        public System.Security.Cryptography.Xml.CipherReference CipherReference { get { throw null; } set { } }
-        public byte[] CipherValue { get { throw null; } set { } }
+        [System.Diagnostics.CodeAnalysis.DisallowNullAttribute]
+        public System.Security.Cryptography.Xml.CipherReference? CipherReference { get { throw null; } set { } }
+        [System.Diagnostics.CodeAnalysis.DisallowNullAttribute]
+        public byte[]? CipherValue { get { throw null; } set { } }
```

Comment:
double check this can be null given constructor annotation. if none of the code expects it might be worth filing an issue on ctor not throwing (same for CipherReference)

Reply:
That one's easy: `byte[] ret = new CipherData().CipherValue;`, `ret` is `null`.

Reply:
fair enough, I didn't notice parameterless ctor